doc/modules/model_evaluation.rst

```diff
@@ -136,13 +151,13 @@ measuring a prediction error given ground truth and prediction:
 - functions ending with ``_score`` return a value to
   maximize, the higher the better.
 
-- functions ending with ``_error`` or ``_loss`` return a
+- functions ending with ``_error``, ``_loss``, or ``_deviance`` return a
   value to minimize, the lower the better.  When converting
   into a scorer object using :func:`make_scorer`, set
   the ``greater_is_better`` parameter to ``False`` (``True`` by default; see the
   parameter description below).
 
-Metrics available for various machine learning tasks are detailed in sections
+Metrics available for various machine learning tasks are detailed in the table
```

Comment:
I think that the previous formulation was correct: we have specific sections below. If we want to refer to the tables, it will be "the table above".

Reply:
Resolving this comment since I like the proposal of @ogrisel